ctriface/failing_test.go

```diff
@@ -35,6 +37,22 @@ import (
 )
 
 func TestStartSnapStop(t *testing.T) {
+	orch := NewOrchestrator(regular.NewRegOrchestrator(
+		"devmapper",
+		"",
+		"fc-dev-thinpool",
+		"",
+		10,
+		regular.WithTestModeOn(true)),
+	)
+
+	vmID := "2"
+	revisionID := "myrev-2"
+
+	startSnapStop(t, orch, vmID, revisionID)
+}
+
+func startSnapStop(t *testing.T, orch *Orchestrator, vmID, revisionID string) {
```

Comment:
This was so I could easily pass the orchestrator for the old and new snapshots to the same function. However, since I re-unified both again into the same orchestrator I reverted this change as suggested.